### ISSUE TITLE
about training 

### ISSUE THREAD

**Issue by myy-imp** [MIC-DKFZ/nnUNet#914]
Dear Fabian, I have  a question about the training. I've already trained the network 1000 epochs. But I want to obtain the result at 500 epochs. If I set the self.max_num_epochs as 500, then I retraining the network using -c, Can I get the model at 500 epochs? Thank you!

**Comment by FabianIsensee** [comment]
Unfortunately you will have to train from scratch again. Maybe model_best is what you are interested in?
Best,
Fabian

**Comment by myy-imp** [comment]
Thank you for your reply!

I found an interesting thing. 

At first, I trained the net with the max_num_epochs as 1000. When the network trained to 512 epochs, I stopped the training artificially. 

Then I modified the max_num_epochs as 500, and retrained the network with -c. And the network began prediction directly without training. I stopped the training artificially again.​

Then I modified the max_num_epochs as 502,and retrained the network with -c.The network started training at 501 epoch.​​

So, if I modified the max_num_epochs as 500, can I get the training result at 499 epoch?

I also have another question. I ran the training with the DDP using two identical GPUs. And when the training was finished and the prediction was began, it always reported errors。 The errors is shown belows. Can you give some advice about this error?
![image](https://user-images.githubusercontent.com/42831398/149857297-9abaf430-4ec8-41bd-a90c-7352c7100289.png)

**Comment by myy-imp** [comment]
When training, can I skip the validation process?  In other words, do not execute function trainer.validate in run_training_DDP.py. Will it affect the inference process and result?

**Comment by FabianIsensee** [comment]
Honestly I don't know what caused the error and if you find out I would greatly appreciate what it is ^^ In general I do not recommend DDP training with nnU-Net because, as the documentation says, it's experimental and you should NOT expect it to work out of the box.
You can of course just skip the validation. This is not recommended however because you then don't get any scores for the validation set. If you continue to have this error you can also just set the number of GPUs to 1 and it should run.

I am not sure what you are doing with nnU-Net and why you are using DDP over regular training. I just want to highlight that my recommendation is to first run nnU-Net in its intended form (single GPU training!) and without modifications before proceeding with anything else. The standard nnU-net should be your baseline
